dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/IntegratorOrFirstOrder.mo

```diff
@@ -13,24 +13,26 @@ within Dynawo.NonElectrical.Blocks.NonLinear;
 */
 
 block IntegratorOrFirstOrder "Switch between two operators : integrator, first-order filter"
-
   import Modelica;
   import Modelica.Blocks.Interfaces;
   import Dynawo.Types;
 
-  extends Modelica.Blocks.Icons.PartialBooleanBlock;
+  extends Modelica.Blocks.Icons.Block;
 
   parameter Types.PerUnit K = 1 "Integrator gain";
   parameter Types.Time tFilter "First-order time constant in s";
   parameter Real Y0 "Value of y at initial time" annotation(
     Dialog(group="Initialization"));
 
+  //Input variables
```

Comment:
I wouldn't put it for base blocks like this

Reply:
Understood. I have done it in BaseRSFlipFlop, do you want me to remove the headings ?

Reply:
Yes, please